code/API_definitions/connected-network-type.yaml

```
@@ -0,0 +1,405 @@
+openapi: 3.0.3
+info:
+  title: Connected Network Type
+  description: |
+    This API provides the customer with the ability to query to which Mobile Communication Technology it is connected to.
+
+    # Introduction
+
+    ## Connected Network Type
+
+    API consumer is able to inquire device's connected network type.
```

Some additional text with an example implementation.
```suggestion
    The API consumer is able to query the device's connected network type. The API provider can determine this in various ways. For example, the network type can be inferred from the specific radio cell to which the device is connected, as radio cells only support a single technology. It is important to note that the API does not query the device directly.
```